docs/svelte-testing-library/intro.mdx

```diff
@@ -40,5 +40,3 @@ Introduction][dom-solution-explainer] for a more in-depth explanation.
 
 1.  A test runner or framework.
 2.  Specific to a testing framework.
-
-We recommend Vitest as our preference.
```

Comment:
This felt odd to add it here under `This Solution - What this library is not` (this is also mentioned in `setup`).
I can revert this if that's preferable